a/aliyungo/aliyungo_ubi_8.3.sh

```diff
@@ -0,0 +1,62 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : aliyungo 
+# Version               : master
+# Source repo           : https://github.com/denverdino/aliyungo 
+# Tested on             : UBI 8.3
+# Script License        : Apache License, Version 2 or later
+# Passing Arguments     : Passing Arguments: 1.Version of package,
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Arumugam N S<asellappen@yahoo.com>/Priya Seth<sethp@us.ibm.com>
+#
+# Disclaimer            : This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=master
+else
+  export VERSION=$1
+fi
+
+if [ -d "aliyungo" ] ; then
+  rm -rf aliyungo
+fi
+
+# Dependency installation
+sudo dnf install -y git golang 
+# Download the repos
+git clone https://github.com/denverdino/aliyungo 
+
+
+# Build and Test aliyungo 
+cd aliyungo 
+git checkout $VERSION
+ret=$?
+if [ $ret -eq 0 ] ; then
+ echo "$VERSION found to checkout "
```

Comment:
VERSION doesn't seem to be displayed in the test output. Pl. check

Reply:
yes.found the issue recently for other script hence modified couple of scripts where we used $Version instead of $VERSION ,I saw most of us having the same issue  ,instead of $VERSION we used $Version.